x-pack/plugins/security/public/management/roles/edit_role/privileges/es/elasticsearch_privileges.tsx

```diff
@@ -162,29 +159,48 @@ export class ElasticsearchPrivileges extends Component<Props, {}> {
             {this.learnMore(docLinks.links.security.indicesPrivileges)}
           </p>
         </EuiText>
-
         <IndexPrivileges
+          indexType="indices"
+          indexPatterns={indexPatterns}
           role={role}
           indicesAPIClient={indicesAPIClient}
           validator={validator}
-          indexPatterns={indexPatterns}
           license={license}
           onChange={onChange}
           availableIndexPrivileges={builtinESPrivileges.index}
           editable={editable}
         />
+        <EuiSpacer />
+        <EuiSpacer />
 
-        {editable && (
-          <>
-            <EuiHorizontalRule />
-            <EuiButton iconType={'plusInCircle'} onClick={this.addIndexPrivilege}>
-              <FormattedMessage
-                id="xpack.security.management.editRole.elasticSearchPrivileges.addIndexPrivilegesButtonLabel"
-                defaultMessage="Add index privilege"
-              />
-            </EuiButton>
-          </>
-        )}
+        <EuiTitle size="xs">
+          <h3>
+            <FormattedMessage
+              id="xpack.security.management.editRole.elasticSearchPrivileges.remoteIndexPrivilegesTitle"
+              defaultMessage="Remote index privileges"
+            />
+          </h3>
+        </EuiTitle>
+        <EuiSpacer size="s" />
+        <EuiText size="s" color="subdued">
+          <p>
+            <FormattedMessage
+              id="xpack.security.management.editRole.elasticSearchPrivileges.controlAccessToRemoteClusterDataDescription"
+              defaultMessage="Control access to the data in remote clusters. "
+            />
+            {this.learnMore(docLinks.links.security.indicesPrivileges)}
```

Comment:
Would we want a more specific doc link in the near future? I don't think more appropriate page exists at the moment, but a page that links to both the remote cluster config page and the index privileges page could be an option.